blocks/social/social.js

```diff
@@ -10,7 +10,7 @@ export default async function decorate(block) {
     const a = x.querySelector('a');
     const span = document.createElement('span');
     const newAnchor = document.createElement('a');
-    newAnchor.href = a.href;
+    newAnchor.href = a.href.replace(/%5C/g, ''); // Replacing extra backslash which is getting appended
```

Comment:
Is this string only appended or can it also appear in the middle?
Wasn't it only happening when the next character was an `&`? 
Just wondering if it makes sense to make the regex a little more specific (e.g. to include the `&`) etc.

Also would it be better to use replaceAll() instead of replace()?

Reply:
Not sure how, but i am getting %5c%5c& pattern in the href. So, i can change the pattern as of now